### ISSUE TITLE
Section 508 502.3.1: (Idle/link 'http://aka.ms/exporting') An onscreen element must not have a null BoundingRectangle property.

### ISSUE THREAD

**Issue by tgugler** [wixtoolset/issues#6361]
If this issue is a bug:

* Which version of WiX are you building with?

> 3.14.0.1119

* Which version of Visual Studio are you building with (if any)?

> Visual Studio 2017

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> 1.0.0.4

* Which version of .NET are you building with?

> Not applicable

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> Not applicable

* Describe the problem and the steps to reproduce it.

User Experience:
If, An onscreen element have a null BoundingRectangle property. , then the AT users  might get affected while navigating on the screen controls and unable to perform its function.

Pre Requisite:
Download exe file from https://www.microsoft.com/en-us/download/details.aspx?id=54688

Repro Steps:
Launch the "NPS Extension For Azure MFA" setup file.
Navigate all the controls of "MICROSOFT SOFTWARE LICENSE TERMS" popup window.
Run Accessibility Insights for windows and verify the issue.

Actual Result :
An onscreen element have a null BoundingRectangle property.

Expected Result:
An onscreen element must not have a null BoundingRectangle property.

Note: 
The same issue repros with others elements of the screen. Attachment is attached.

**Comment by tgugler** [comment]
![MAS4 2 1_An on screen element must not have a null BoundingRetangle property](https://user-images.githubusercontent.com/12263599/108531449-8d34cb00-728b-11eb-9e15-3655c50b9983.png)
![1284838-pic](https://user-images.githubusercontent.com/12263599/108531450-8dcd6180-728b-11eb-8009-91718a007e2e.png)

**Comment by BMurri** [comment]
My best guess? Each one of those links is embedded in the licence document. One question and one comment:

If one of the URLs is scrolled into view, would it then aquire a non-null BoundingRectangle?

This would appear to be an issue with the control itself. Are we using an obsolete version of that control?

**Comment by rseanhall** [comment]
The control was updated in v4 for https://github.com/wixtoolset/issues/issues/5843

**Comment by BMurri** [comment]
Then these two could be closed, unless we wanted to apply the same fix in v3.
